front/app/containers/HomePage/SignedInHeader/HeaderImage.tsx

```diff
@@ -32,24 +32,15 @@ const StyledImage = styled(Image)`
   height: auto;
 
   ${media.tablet`
-    &.objectFitCoverSupported {
-      width: 100%;
-      height: 100%;
-      object-fit: cover;
-    }
-
-    &:not(.objectFitCoverSupported) {
-      width: auto;
-      height: 100%;
-    }
```

Comment:
Not needed anymore since we don't need to support IE11 anymore. https://caniuse.com/?search=object-fit